recipes/poco/all/conanfile.py

```diff
@@ -158,6 +169,9 @@ def _configure_cmake(self):
         return self._cmake
 
     def build(self):
+        if self.options.enable_data_sqlite:
+            if self.options["sqlite3"].threadsafe == 0:
+                raise ConanInvalidConfiguration("sqlite3 must be built with threadsafe enabled")
```

Comment:
Should this be in configure?

Reply:
Ideally, yes, because it is checking options.
No, because the options of dependent packages is `None` in the `configure` method.
I don't know whether that is a bug or intended feature.

Reply:
@danimtb @uilianries Where/how should we check the options of our dependencies?

Reply:
that's the correct way. The graph is computed after configure, so you can only check after configure.